### ISSUE TITLE
Hide tags from detail page

### ISSUE THREAD

**Issue by agneevX** [sct/overseerr#3350]
### Description

The ability to hide tags from the detail page

Apart from it not being potentially useful to some users and causing visual clutter (especially on mobile screens), it also gives away spoilers for the content.

<img width="912" alt="image" src="https://user-images.githubusercontent.com/19761269/220923180-364e4c21-4452-4ac1-8513-274c489406ca.png">

### Desired Behavior

The ability to hide tags

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by alindsharmasimply** [comment]
Hey @agneevX , do you want a toggle button in place of the tags with a label 'Show Tags'? The label would change to 'Hide tags' when the button is toggled. By default, the setting of the button would stay 'Show Tags'.

I'm starting on this.

**Comment by alindsharmasimply** [comment]
Also, kindly brief me the steps for reaching the above page. Currently I can see the localhost:3000/setup page. How should I navigate to the above page? Do we have a test user for signing-in?

**Comment by sambartik** [comment]
I think it would make more sense on mobile layout to show a few tags and hide the rest with the ability to show all of them after a click. That would help with the cluttering. I don't think tags give away spoilers. If so, you might very well want to hide the description as well.

**Comment by danshilm** [comment]
Another option I had in mind is to have a button to hide/show the tags and having that preference saved in the browser storage. That would require no additional permissions, whether it's global or user permissions.

We do the same thing with filters, the sort order and the page size on the Requests page and others.

**Comment by sambartik** [comment]
Yeah, I see your point. I wouldn't object to resolving it this way.
